reference/format/datagrid.rst

```diff
@@ -0,0 +1,166 @@
+Datagrid
+========
++-----------+------------------+
+| Filename  | ``datagrid.yml`` |
++-----------+------------------+
+| Root Node | ``datagrid``     |
++-----------+------------------+
+
+The datagrid configuration is a large file that is a map of datagrid names mapped to options that
+configure their behavior:
+
+.. _reference-datagrid-columns:
+
+``actions`` (**type**: ``map``)
```

Comment:
All of your parameters in all files are sorted alphabetically, but parameters in real config files does not sorted alphabetically. I think, you should save the parameter's order as in real config files. For example, it is a little bit confuse me when i does not see the main `source ` parameter for datagrid yml structure at the top of page.
What do you think?